File: SelfSchoolBE/SelfSchoolDAL/Entities/Parent.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SelfSchoolDAL.Entities
{
    public class Parent
    {
        [Key]
        public int idParent { get; set; }
        public string loginParent { get; set; }
        public string passwordParent { get; set; }
        public string nameParent { get; set; }
        public string surnameParent { get; set; }
        public long birthdayParent { get; set; }
        public string emailParent { get; set; }
        public string phoneParent { get; set; }
    }
}

File: SelfSchoolFE/src/app/classes/parent.ts
export class Parent {
  private readonly _idParent: number = 0;
  private readonly _loginParent: string = '';
  private readonly _passwordParent: string = '';
  private readonly _nameParent: string = '';
  private readonly _surnameParent: string = '';
  private readonly _birthdayParent: number = 0;
  private readonly _emailParent: string = '';
  private readonly _phoneParent: string = '';

  constructor(idParent: number, loginParent: string, passwordParent: string, nameParent: string,
              surnameParent: string, birthdayParent: number, emailParent: string, phoneParent: string) {
    this._idParent = idParent;
    this._loginParent = loginParent;
    this._passwordParent = passwordParent;
    this._nameParent = nameParent;
    this._surnameParent = surnameParent;
    this._birthdayParent = birthdayParent;
    this._emailParent = emailParent;
    this._phoneParent = phoneParent;
  }

  public get idParent(): number {
    return this._idParent;
  }

  public get loginParent(): string {
    return this._loginParent;
  }

  public get passwordParent(): string {
    return this._passwordParent;
  }

  public get nameParent(): string {
    return this._nameParent;
  }

  public get surnameParent(): string {
    return this._surnameParent;
  }

  public get birthdayParent(): number {
    return this._birthdayParent;
  }

  public get emailParent(): string {
    return this._emailParent;
  }

  public get phoneParent(): string {
    return this._phoneParent;
  }
}
